csharp/Platform.Collections/IDictionaryExtensions.cs

```diff
@@ -4,17 +4,91 @@
 
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
 
-namespace Platform.Collections
+namespace Platform.Collections 
 {
+    /// <summary>
+    /// <para>Represents a set of extension methods <see cref="IDictionary{TKey, TValue}"/> array.</para>
+    /// <para>Представляет набор методов расширения <see cref="IDictionary{TKey, TValue}"/>.</para> 
+    /// </summary>  
     public static class IDictionaryExtensions
     {
+        /// <summary>    
+        /// <para> 
+        /// Returns the value associated with the passed parameters  
```

Comment:
Do not break lines.
Use `Gets` in english version according [msdn](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.item?view=net-5.0)
`Gets the value associated with the <paramref name="key"/> in the <paramref name="dictionary"/> or default value if it does not exist.`